### DISCUSSION TITLE
Proposta de alteração do Código de Conduta

### DISCUSSION THREAD

**Discussion by anicelysantos** [apyb/comunidade#53] [Geral]
Olá!

Participei da Equipe de Resposta da Python Brasil 2022 e durante nossa leitura conjunta do CDC para alinhar as interpretações, percebemos a importância de deixar explícito algumas informações. Com isso, venho propor alguns adendos no Código atual, que são:

 - Mais descrição no ponto 1 do código
 **onde se ler:** Não é tolerado nenhum tipo de assédio, discriminação inapropriada ou humilhação pública
 **Passar a ser:** Não é tolerado nenhum tipo de assédio independente de identidade de gênero, sexo, orientação sexual, deficiência, aparência física, composição corporal, raça, classe social, religião e qualquer representação de diversidade;

- Mais descrição sobre a quem está sujeita as regras:
**onde se ler**: Toda pessoa presente no evento, independente do seu papel, está sujeita a estas regras;
**Passar a ser**: Toda pessoa presente no evento, independente do seu papel, está sujeita a estas regras, a saber: participantes, ministrantes de atividades, organização, equipe de resposta, patrocinadores, prestadoras de serviço;

- Devido a continuidade da Pandemia, incluir mais esse parágrafo (que pode ser aplicado a outras coisas também, mas o contexto de inclusão seria para o uso de máscara):
**Proposta de inclusão**: 4. Não é permitido nenhum tipo de ação que coloque em risco a segurança de outra pessoa.

Muita gente ainda acha que o CDC se aplica a tudo no evento, desde grupos não oficiais a confraternizações não oficiais. Diante disso, proponho a essa inclusão no final:

*O Código de Conduta se aplica apenas as atividades da programação oficial do evento. PyBar, Happy Hours privados ou quaisquer atividades semelhantes não estão cobertas pelo Código de Conduta.*

No geral, o novo CDC ficaria assim:

O evento Python Brasil é um ambiente amistoso, de boa convivência, inclusivo e livre de intimidação, onde todas as pessoas são bem-vindas e a civilidade é exigida.

Com esta finalidade, a organização do evento conta com uma Equipe de Resposta que atua para garantir um ambiente com essas qualidades.

Por isso:

1. Não é tolerado nenhum tipo de assédio independente de identidade de gênero, sexo, orientação sexual, deficiência, aparência física, composição corporal, raça, classe social, religião e qualquer representação de diversidade;
2. Não é tolerado o descumprimento das leis brasileiras;
3. Toda pessoa presente no evento, independente do seu papel, está sujeita a estas regras, a saber: participantes, ministrantes de atividades, organização, equipe de resposta, patrocinadores, prestadoras de serviço;
4. Não é permitido nenhum tipo de ação que coloque em risco a segurança de outra pessoa.

Desta forma, entendemos que:

* Assédio é a ação de insistir, perseguir ou coagir outra pessoa a um comportamento involuntário.
* Discriminação inapropriada é o ato de separar, injuriar ou humilhar alguém promovendo sua exclusão por atributo particular da mesma.
* Humilhação pública é o ato de submeter, rebaixar, ridicularizar ou promover o vexame de outro publicamente.

O público alvo do evento também inclui crianças e adolescentes, por isso buscamos manter um ambiente apropriado para todas as faixas etárias. Sendo assim, linguagem e imagens sexualizadas não são adequados para palestras e ações promocionais de patrocinadores.

Se você se sentiu assediado, discriminado indevidamente ou humilhado, ou presenciou alguma destas atitudes, por favor entre em contato com a Equipe de Resposta.

Havendo um relato de violação destes princípios, a Equipe de Resposta realizará a devida análise e, quando necessário, tomará as ações para impedir a reincidência. Estas ações podem, mas não se restringem nem implicam em ir desde uma conversa em busca da retratação até um convite a se retirar do evento.

O Código de Conduta se aplica apenas as atividades da programação oficial do evento. PyBar, Happy Hours privados ou quaisquer atividades semelhantes não estão cobertas por este Código.

**Comment by gabubellon** [comment]
Só deixar claro que ele se aplica a locais virtuais como o Discord 
`O Código de Conduta se aplica apenas as atividades da programação oficial do evento e em grupos e forum virtuais administrados pela organização do evento.
Eventos realizados de forma independente e sem relação direta e explicita com a organização do evento , como PyBar, Happy Hours privados, grupos de mensagens e email privados e não oficiais, ou quaisquer atividades semelhantes não estão cobertas pelo Código de Conduta.`

**Comment by gabubellon** [comment]
Sobre o item 4 novo
`Proposta de inclusão: 4. Não é permitido nenhum tipo de ação que coloque em risco a segurança, a saúde e a vida de de uma ou grupo de pessoas`

**Comment by rougeth** [comment]
Sobre o primeiro, eu ainda deixaria genérico na primeira parte (o que está abaixo de *Por isso:*), e incluiria mais detalhes no segunda parte, a partir de *Desta forma, entendemos que:*, por exemplo: 

```diff
+ - Assédio é a ação de insistir, perseguir ou coagir outra pessoa a um comportamento involuntário, independente de identidade de gênero, sexo, orientação sexual, deficiência, aparência física, composição corporal, raça, classe social, religião e qualquer representação de diversidade;
- - Assédio é a ação de insistir, perseguir ou coagir outra pessoa a um comportamento involuntário.
```

Sobre o segundo ponto, também deixaria genérico na primeira parte, e na segunda, incluiria exemplos.

Sobre o terceiro ponto, que situação o item `Não é tolerado o descumprimento das leis brasileiras;` não comporta? Em que situação é permitido por lei colocar em risco a vida das pessoas? Especificamente sobre uso de máscaras, poderíamos obrigar as pessoas a usarem?

No último ponto, a questão de não estar claro onde o CdC pode ser aplicado, vejo mais como um problema de divulgação da organização. Acho ruim limitar no CdC quais são as atividades oficias, porque todo ano o evento muda, o que era oficial para um, pode não ser para outro.

---

Para quem tiver curiosidade, essa foi a conversa na lista de e-mails da associação quando esse código de conduta foi criado e anunciado: https://groups.google.com/g/apyb-associados/c/RXR_0gosYbY/m/T8x0z4p2AAAJ

**Comment by anicelysantos** [comment]
Concordo ali no ponto de ampliar a explicação do assédio, ao invés de vir logo em cima, e concordo também na sugestão pra o segundo ponto ser embaixo. A ideia era deixar o mais explícito possível.

Sobre máscaras, acho que talvez não consiga enquadrar dentro desse ponto das leis brasileiras, mas talvez juridicamente podemos ter algo que respalde ser uma obrigatoriedade do evento (quando for o caso) e que o fato do participante estar ali, já concorda em obedecer, dado que no ato do ingresso deve vir (não lembro se vem sempre) o checkbox de que a pessoa concorda com o CdC.

Sobre os espaços onde é aplicado, sim, concordo com pouca divulgação do CdC. Acho que pensar uma maneira de ficar a importância dele ajude no processo sem precisar que seja incluído alguma coisa.
